src/pages/MovieList.jsx

```diff
@@ -0,0 +1,170 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import '../styles/movielist.css'
+import Magnify from '../../public/search.svg'
+import Ig from '../../public/instagram.svg'
+import Git from '../../public/github.svg'
+import Lk from '../../public/linkedin.svg'
+import Logo from '../../public/logo.png'
+
+function MovieList() {
+  const [list, setList]= useState([])
+  const [page, setPage]= useState(1)
+  const [expand, setExpand]= useState(0)
+  const [clicked, setClicked]= useState(false)
+  const [load, setLoad]= useState(true)
+
+  function getExpanded(id) {
+
+    if(clicked === true) {
+      setClicked(false)
+      setExpand(0)
+    }
+    else {
+      setClicked(true)
+      setExpand(id)
+    }
+  }
+
+  function getData() {
+    axios.get(`https://api.themoviedb.org/4/list/1?page=${page}&api_key=9e89dc1810e00461d6f59011e04175ed`).then((movies) => {
+      setTimeout(() => setLoad(false), 1000)
+      setList(movies.data.results)
+    })
+  }
+
+  function getSearched(value) {
+   
+    if (value !== '') {
+      axios.get(`https://api.themoviedb.org/3/search/movie?query=${value}&&api_key=9e89dc1810e00461d6f59011e04175ed`).then((filter) => {
+      setList(filter.data.results)
+      })
+    }
+    else if (value.length === 0) {
+      getData()
+    }
+  }
+
+  useEffect(() => {
+    getData()
+  }, [page])
+  return (
+   <>
+    {load ? (
+      <div className="container-svg">
+        <svg className='svg' viewBox='0 0 50 50'>
+          <circle
+            cx={25}
+            cy={25}
+            r={20}
+          />
+        </svg>
+      </div>
+    ) 
+    : 
+    (
+      <>
+        <div className="content">
+          <nav>
+            <ul>
+              <li>
+                <Link className='link' to='/'>
+                  Home
+                </Link>
+              </li>
+              <li
+              onClick={() => {
+
+                if (page !== 1) {
+                  setPage(1)
+                  setLoad(true)
+                } else return 0
+                
+              }}
+              >
+                Page 1              
+              </li>
+              <li
+              onClick={() => {
+                if (page !== 2) {
+                  setPage(2)
+                  setLoad(true)
+                } else return 0
+              }}
+              >
+                Page 2
+              </li>
+              <li
+              onClick={() => {
+                if (page !== 3) {
+                  setPage(3)
+                  setLoad(true)
+                } else return 0
```

Comment:
maybe you can separate these onClick functions with a handler function that recieves the page number as params. instead of making the function one by one like this.